events/ome-community-meeting-2020/workshops/index.html

```diff
@@ -3,43 +3,19 @@
 nav-workshops: active
 title: Day 3 - Thursday, May 28
 subheader: Learn more about using OMERO through our practical workshops
-schedule:
-    - time: "11:45 - 12:10"
-      type: Workshop
-      title: "OMERO Analysis Workflows"
-      details: with Jean-Marie Burel and Petr Walczysko
-    - time: "12:10 - 12:35"
-      type: Workshop
-      title: "QuPath"
-      details: with Pete Bankhead
-    - time: "12:35 - 13:00"
-      type: Workshop
-      title: "Napari"
-      details: with Nicholas Soforniew
-    - time: "13:00 - 13:25"
-      type: Workshop
-      title: "OME Ontology and the Open Science World"
-      details: with Norio Kobayashi
-    - time: "12:35 - 13:30"
-      type:
-      title: "Lunch/Posters"
-      details: Details to follow soon.
-    - time: "14:00 - 14:30"
-      type: "Invited Talk #1"
-      title: TBD
-      details: Details to follow soon.
-    - time: "15:00 - 16:00"
-      type: "Invited Talk #2"
-      title: TBD
-      details: Details to follow soon.
-    - time: "16:00 - 16:30"
-      type:
-      title: Coffee
-      details: Time to refill that coffee cup, grab a snack, and chat up one of the speakers.
-    - time: "16:30 - 17:30"
-      type:
-      title: Analysis
-      details: Details to follow soon.
 ---
 
-{% include program_agenda.html %}
+<div class="row">
+    <div class="small-12 medium-6 medium-offset-3 columns">
+        <div class="card card-support">
+            <div class="card-section">
+                <i class="border-green icon-green fa fa-comments-o fa-2x"></i>
+                <h4>Workshops TBD</h4>
+                <p class="card-caption">
+                    Workshops are still to be determined but if you are interested in running one, please reach out to <a href="mailto:J.Matthew@dundee.ac.uk?&cc=j.r.swedlow@dundee.ac.uk?&subject=I want to run a workshop!">June Matthew and Jason Swedlow</a>.
+                </p>
+                <p class="text-center"><a href="mailto:J.Matthew@dundee.ac.uk?&cc=j.r.swedlow@dundee.ac.uk?&subject=I want to run a workshop!" class="btn-blue button small">I want to run a workshop!</a></p>
```

Comment:
This nice trick with mail to June and cc jason does not work on some setups. For example, my default email for the OS is not the one I really use. When I try to copy the email address by right-clicking on it (in order to paste it into my real email app), only June's address gets copied and Jason's is completely lost for me - I would suggest not putting Jason as a cc, but instead as the second addressee would fix the issue.